### PR TITLE
Fixed typos and grammar

### PR DESCRIPTION
**Pull request by RahulJanagouda**

### CODE REVIEW

**Comment by RahulJanagouda** [comment]
signed

File: style.md [hunk 1]
```diff
@@ -665,13 +665,13 @@ Each type variable is named in one of two styles:
 
 ## Camel case
 
-Sometimes there is more than one reasonable way to convert an English phrase into camel case, such as when acronyms or unusual constructs like "IPv6" of "iOS" are present. To improve predictability, use the following scheme.
+Sometimes there is more than one reasonable way to convert an English phrase into camel case, such as when acronyms or unusual constructs like "IPv6" or "iOS" are present. To improve predictability, use the following scheme.
 
 Beginning with the prose form of the name:
 
- 1. Convert the phrase to plain ASCII and remove any apostrophes. For example, "Müller's algorithm" might become "Muellers algorithm".
+ 1. Convert the phrase to plain ASCII and remove any apostrophes. For example, "Müller's algorithm" might become "Mueller's algorithm".
```
Comment (JakeWharton) [inline style.md]: The rule explicit says to remove apostrophes so this is incorrect.

File: style.md [hunk 2]
```diff
@@ -665,13 +665,13 @@ Each type variable is named in one of two styles:
...
 
- 2. Divide this result into words, splitting on spaces and any remaining punctuation (typically hyphens).
+ 2. Divide this result into words, splitting into spaces and any remaining punctuation (typically hyphens).
```
Comment (JakeWharton) [inline style.md]: "Splitting into spaces" does not convey the action that you should take. Furthermore, this wording matches the [Java style guide](https://google.github.io/styleguide/javaguide.html).

File: style.md [hunk 3]
```diff
@@ -583,7 +583,7 @@ package com.example.deep_space
 
 ## Type names
 
-Class names are written in PascalCase and are typically nouns or noun phrases. For example, `Character` or `ImmutableList`. Interface names may also be nouns or noun phrases (for example, `List`), but my sometimes be adjectives or adjective phrases instead (for example `Readable`).
+Class names are written in PascalCase and are typically nouns or noun phrases. For example, `Character` or `ImmutableList`. Interface names may also be nouns or noun phrases (for example, `List`), but sometimes be adjectives or adjective phrases instead (for example `Readable`).
```
Comment (JakeWharton) [inline style.md]: This was a typo and is now fixed on master to correctly say "may", so please revert.

File: style.md [hunk 4]
```diff
@@ -262,14 +262,14 @@ The prime directive of line-wrapping is: prefer to break at a **higher syntactic
  4. A comma (`,`) stays attached to the token that precedes it.
  5. A lambda arrow (`->`) stays attached to the argument list what precedes it.
 
-Note: The primary goal for line wrapping is to have clear code, _not necessarily_ code that fits in the smallest number of lines.
+Note: The primary goal for line wrapping is to have a clear code, _not necessarily_ code that fits in the smallest number of lines.
```
Comment (JakeWharton) [inline style.md]: This does not read correctly and should be reverted.

File: style.md [hunk 5]
```diff
@@ -333,7 +333,7 @@ A single blank line appears:
 
  1. _Between_ consecutive members of a class: properties, constructors, functions, nested classes, etc.
 
-     * **Exception**: A blank line between two consecutive properties (having no other code between them) is optional. Such blank lines are used as needed to create logical groupings of properties and associate properties with their backing property, if present.
+     * **Exception**: A blank line between two consecutive properties (having no other code between them) is optional. Such blank lines are used as needed to create logical groupings of properties and associate properties with their backing property if present.
```
Comment (JakeWharton) [inline style.md]: This seems incorrect. I believe "if present" is a participle phrase which requires a comma.

**Comment by RahulJanagouda** [comment]
😳 Whoa... reverting

**Pull request closed without merging**